src/plots/cartesian/set_convert.js

```diff
@@ -417,6 +417,10 @@ module.exports = function setConvert(ax) {
     ax._min = [];
     ax._max = [];
 
+    // copy ref to fullLayout.separators so that
+    // methods in Axes can use it w/o having to pass fullLayout
+    ax.separators = fullLayout.separators;
```

Comment:
**Important:** link `fullLayout.separators` to full axis object, so that `Axes` method can use it without having to know anything about `fullLayout`.